Makefile

```diff
@@ -573,3 +573,11 @@ search_interfaces: ## Greps and outputs all of the structs in the project (exclu
 .PHONY: search_protos
 search_protos: ## Finds all of the proto files in the project (excluding vendor)
 	find . -name "*.proto" -not -path "./vendor/*"
+
+.PHONY: ggshield_secrets_scan
+ggshield_secrets_scan: ## Scans the project for secrets using ggshield
+	ggshield secret scan path --recursive .
+
+.PHONY: ggshield_secrets_add
+ggshield_secrets_add: ## Ignore recently foudn secrets using ggshield
+	ggshield secret ignore --last-found
```

Comment:
Doesn't this effectively disable `ggshield`? i.e. it won't complain even if we actually include a secret in the diff by mistake.

Reply:
Correction to my comment above: it would perhaps make it a bit too easy to miss a secret by ignoring it. Perhaps there is a mechanism to add a comment to the line that could be interpreted as a secret to get `ggshield` to ignore it.

Reply:
Wdyt of the updated comment?